docs/_plugins/search_index.rb

```diff
@@ -218,6 +225,18 @@ def execute
       },
     },
   )
+  exists =  client.indices.exists index: 'models'
```

Comment:
Please don't create the index here. The index should contain more fields.

Reply:
Currently, we are dynamically adding the fields.. This will only add the `keyword` type to the `url` field. When we bulk insert, all other fields will be created dynamically

Reply:
Dynamic fields won't be created correctly.

Reply:
Adding the full mapping